src/thing_traps.c

```diff
@@ -382,7 +382,7 @@ void activate_trap_shot_head_for_target90(struct Thing *traptng, struct Thing *c
         shotng->veloc_push_add.y.val += cvect.y;
         shotng->veloc_push_add.z.val += cvect.z;
         shotng->state_flags |= TF1_PushAdd;
-        shotng->byte_16 = trapstat->field_1B;
+        shotng->hit_type = trapstat->hit_type;
```

Comment:
`shotng` is another branch of union. Is it still hit_type?

Reply:
Shots and effects both use hit types yes. The same value in determining what can be damaged/moved by the effects/area damage/shots.